CHANGES.md

```diff
@@ -24,6 +24,18 @@ OpenSSL 3.1
 
 ### Changes between 3.0 and 3.1 [xx XXX xxxx]
 
+ * Add new function and CTRL code for TLS/SSL fingerprinting improvement.
+
+   New CTRL codes and macros:
+
+      SSL_CTRL_GET_IANA_GROUPS - SSL_get0_iana_groups()
+
+   New functions:
+
+      SSL_client_hello_get_raw_extensions()
+
+   *Phus Lu*
```

Comment:
I don't see much precedent for having this much internal vertical whitespace in a CHANGES entry.  The prevailing style would have the content condensed into a single prose paragraph like:

```
 * Add new SSL APIs to aid in efficiently implementing TLS/SSL fingerprinting.  The
  SSL_CTRL_GET_IANA_GROUPS control code, exposed as the S
  SL_get0_iana_groups() function-like macro, retrieves the list of supported groups
  sent by the peer, and the function SSL_client_hello_get_raw_extensions() populates
  a caller-supplied array with the list of extension types present in the ClientHello, in
  order of appearance.
```

Reply:
done with thanks.